app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -570,7 +575,18 @@ class CommonTriageQuestionsController @Inject() (
                 representeeAnswers = None,
                 examplePropertyDetailsAnswers = None,
                 yearToDateLiabilityAnswers = None,
-                supportingEvidenceAnswers = None
+                supportingEvidenceAnswers = None,
+                exemptionAndLossesAnswers =
+                  if (furtherReturn)
```

Comment:
here and on the other cases below the whole exemptionsAndLosses section needs to remain intact to preserve the default value for AEA